apps/hobbies-helsinki/next-i18next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  i18n: {
+    defaultLocale: 'default',
```

Comment:
I suspect that the translations are broken because the `defaultLocale` might be `fi` in `common-events`or `common-i18n` package.

Reply:
already fixed